sensor_msgs/Image image
aarapsi_intro_pack/ImageStructure data
std_msgs/Header header
